homeassistant/components/thermostat/homematic.py

```diff
@@ -108,21 +144,48 @@ def device_state_attributes(self):
                 "battery": self._battery,
                 "mode": self._mode}
 
+    @property
+    def min_temp(self):
```

Comment:
It can also be set to a temperature (that open/closed functionality is not terribly useful in my opinion).
This is true for all eq3 products.
They also all have an automatic mode with a builtin time program as well as a manual mode where that functionality is disabled, and can be set to a target temperature in both auto and manual mode (which is the functionality that is used by this component).
The mode flag is queried by the component and could potentially be used in automation rules (probably with the time program on the thermostat itself deleted).